accurest-core/src/main/groovy/io/codearte/accurest/dsl/WiremockRequestStubStrategy.groovy

```diff
@@ -27,12 +30,47 @@ class WiremockRequestStubStrategy extends BaseWiremockStubStrategy {
 	private Map<String, Object> buildRequestContent(ClientRequest request) {
 		return ([method    : request?.method?.clientValue,
 		        headers   : buildClientRequestHeadersSection(request.headers)
-		] << appendUrl(request) << appendBody(request)).findAll { it.value }
+		] << appendUrl(request) << appendQueryParameters(request) << appendBody(request)).findAll { it.value }
 	}
 
 	private Map<String, Object> appendUrl(ClientRequest clientRequest) {
-		Object url = clientRequest?.url?.clientValue
-		return url instanceof Pattern ? [urlPattern: ((Pattern)url).pattern()] : [url: url]
+		def urlPath = clientRequest?.urlPath?.clientValue
+		if (urlPath) {
+			return [urlPath: urlPath]
+		}
+		def url = clientRequest?.url?.clientValue
+		return url instanceof Pattern ? [urlPattern: url.pattern()] : [url: url]
+	}
+
+	private Map<String, Object> appendQueryParameters(ClientRequest clientRequest) {
+		def queryParameters = clientRequest?.urlPath?.queryParameters ?: clientRequest?.url?.queryParameters
```

Comment:
Also are you sure that when you're using null safe dereferencing you need to provide ? everywhere? @szpak - I think we went through this that 

```
a?.b?.c?.d
```

can be rewritten to

```
a?.b.c.d
```

?